### PR TITLE
Fix null exception when saving a file

### PR DESCRIPTION
**Pull request by xsombra**
When the file has a fill with a null PatternFill, it throws a
NullReferenceException on XLWorkbook.Save()

### CODE REVIEW

**Comment by igitur** [comment]
Thanks for the contribution. Please reformat your code according to standard Visual Studio conventions. The normal "Format Document" action in Visual Studio should be enough.

File: ClosedXML_Tests/Excel/Saving/SavingTests.cs [hunk 1]
```diff
@@ -575,5 +575,19 @@ public void RemoveExistingInlineStringsIfRequired()
...
+        {
+            // Arrange
+            using (var stream = TestHelper.GetStreamFromResource(TestHelper.GetResourcePath(@"TryToLoad\EmptyFill.xlsx")))
+            using (var wb = new XLWorkbook(stream))
+            using (var ms = new MemoryStream())
+            {
+                // Act
+                wb.SaveAs(ms, false);
+            }
+                // Assert
```
Comment (Pankraty) [inline ClosedXML_Tests/Excel/Saving/SavingTests.cs]: Though I understand that you are checking that `SaveAs` does not throw, test without Assert looks unfinished. Better make you intention explicit:
```
Assert.DoesNotThrow(() => wb.SaveAs(sa, false));
```

**Comment by xsombra** [comment]
Hello. Thank you for your comments. I uploaded the suggested changes.

**Comment by igitur** [comment]
@Pankraty You happy with this PR?

**Comment by Pankraty** [comment]
Absolutely!

**Comment by igitur** [comment]
@xsombra Thanks for the contribution. Please keep them coming.

**Pull request merged**